Shoko.Server/Commands/Interfaces/ICommandProcessor.cs

```diff
@@ -0,0 +1,11 @@
+using Shoko.Commons.Queue;
+using Shoko.Models.Server;
+using Shoko.Server.Server;
+
+namespace Shoko.Server.Commands.Interfaces;
+
+public interface ICommandProcessor
```

Comment:
The idea was that we could more easily update the queue state from the command itself, when it's connected to a ~~queue~~_command processor_.